pulser/simulation/simulation.py

```diff
@@ -252,17 +252,22 @@ def build_operator(self, operations: Union[list, tuple]) -> qutip.Qobj:
                         raise ValueError(
                             f"{operator} is not a valid operator")
                 for qubit in qubits:
-                    if isinstance(qubit, int):
-                        if qubit >= self._size:
-                            raise ValueError(
-                                "The index value is greater than the "
-                                "size of the system")
-                        k = qubit
-                    else:
-                        if qubit not in self._qid_index.keys():
-                            raise ValueError(
-                                f"{qubit} is not a valid qubit")
+                    # if isinstance(qubit, int):
+                    #     if qubit >= self._size:
+                    #         raise ValueError(
+                    #             "The index value is greater than the "
+                    #             "size of the system")
+                    #     k = qubit
+                    # else:
+                    #     if qubit not in self._qid_index.keys():
+                    #         raise ValueError(
+                    #             f"{qubit} is not a valid qubit")
+                    #     k = self._qid_index[qubit]
```

Comment:
You can just delete these instead. If you ever need to retrieve this, you can just go back to a commit where you had it